src/components/Summary/RatingTrends.js

```diff
@@ -50,6 +51,20 @@ function RatingTrends({ activeMeasure, trends, info }) {
   });
 }
 
+const returnStars = (activeMeasure) => {
+  let returnBool = false;
+
+  // Additional stars rules can be added here
+  if (activeMeasure.starRating > 0) {
+    returnBool = true;
+  }
+  if (activeMeasure.denominator > 30) {
```

Comment:
This is handled in the backend already. When the star rating is calculated it checks the denominator. But if you really want it we can keep it.